sale_margin_mrp/models/sale_order.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.onchange("product_id", "product_uom")
+    def product_id_change_margin(self):
+        """Action bom cost computes the standard price from the BoM components.
+        We reproduce here the condition that makes the manual action visible
+        or not in the product form to trigger it automaticaly."""
```

Comment:
```suggestion
        Override. Action bom cost computes the standard price from the BoM components.
        We reproduce here the condition that makes the manual action visible
        or not in the product form to trigger it automaticaly.
```

Nitpicking here: When overwritting, it's better not to overwrite the original method's docstring